soccer/src/soccer/window_evaluator.cpp

```diff
@@ -13,16 +13,17 @@ REGISTER_CONFIGURABLE(WindowEvaluator)
 
 using namespace rj_geometry;
 
-ConfigDouble* WindowEvaluator::angle_score_coefficient;
-ConfigDouble* WindowEvaluator::distance_score_coefficient;
+DEFINE_NS_FLOAT64(windowConfigParamModule, optimization, angle_score_coefficient, 0.7,
+	"Angle score coefficient.");
```

Comment:
Do we not actually use this variable anywhere?

Reply:
We don't. Should I leave it in, since there's a value associated with it?